--- /usr/share/X11/xkb/symbols/us.orig	2024-02-05 02:24:20.000000000 +0100
+++ /usr/share/X11/xkb/symbols/us	2024-05-24 09:58:30.232819468 +0200
@@ -70,6 +70,37 @@
     include "level3(ralt_switch)"
 };
 
+// modified by wasmitnetzen
+// Added German and some Swedish letters to sensible locations, switched y and z
+// Added German quote marks
+// Added degree, section, Euro, mu, thin space (U2009), nobreakspace
+// Added dead_abovering
+
+partial alphanumeric_keys
+xkb_symbols "wasmitnetzen" {
+
+    include "us(euro)"
+    name[Group1]= "English (US, extended by wasmitnetzen)";
+
+    key <CAPS> { [] };
+
+    key <AE04> { [4,      dollar,      section                ] };
+    key <AE06> { [6,      asciicircum, degree, dead_abovering ] };
+    key <AB01> { [y,      Y                                   ] };
+    key <AB07> { [m,      M,           mu,     mu             ] };
+    key <AB08> { [comma,  less,        doublelowquotemark     ] };
+    key <AB09> { [period, greater,     leftdoublequotemark    ] };
+    key <AC01> { [a,      A,           adiaeresis, Adiaeresis ] };
+    key <AC02> { [s,      S,           ssharp,     U1E9E      ] };
+    key <AD03> { [e,      E,           EuroSign               ] };
+    key <AD06> { [z,      Z                                   ] };
+    key <AD07> { [u,      U,           udiaeresis, Udiaeresis ] };
+    key <AD09> { [o,      O,           odiaeresis, Odiaeresis ] };
+    key <AD10> { [p,      P,           aring,      Aring      ] };
+    key <SPCE> { [space,  space,       U2009,      nobreakspace ] };
+    key <PAUS> { [XF86AudioMute, XF86AudioMicMute ] };
+};
+
 
 partial alphanumeric_keys
 xkb_symbols "ibm238l" {
